modules/document_repository/ajax/documentEditUpload.php

```diff
@@ -14,35 +14,27 @@
 if (!$userSingleton->hasPermission('document_repository_view')
     && !$userSingleton->hasPermission('document_repository_delete')
 ) {
-    header("HTTP/1.1 403 Forbidden");
+    http_response_code(403);
     exit;
```

Comment:
Should this have an exit message to let the user know that they do not have permission to access the page, or is that implied?

Reply:
That's what error 403 means, so yes. This function also automatically gives a response message (i.e. "Forbidden" in this case)